feat: allow enabling and disabling compression and decompression separately (#362)
fastify/fastify-compress: README.md

  { global: false }
)
```

> If only compression, or only decompression is desired, you can use the `globalCompression` or `globalDecompression` config flags to turn either off specifically (because the global hook is enabled by default).

Reviewer: ```suggestion
If only compression or decompression is required, set the `globalCompression` or `globalDecompression` config flags to `false` respectively (both are `true` by default).
```
